index.bs

```diff
@@ -559,35 +577,56 @@ Width of the play-area bounds in meters. The bounds are defined as an axis-align
 <dfn attribute for="VRStageParameters">sizeZ</dfn>
 Depth of the play-area bounds in meters. The bounds are defined as an axis-aligned rectangle on the floor. The center of the rectangle is at (0,0,0) in standing-space coordinates. These bounds are defined for safety purposes. Content should not require the user to move beyond these bounds; however, it is possible for the user to ignore the bounds resulting in position values outside of this rectangle.
 
-
-## Navigator Interface extension ## {#interface-navigator}
+## VR Interface ## {#interface-vr}
 
 <pre class="idl">
-partial interface Navigator {
-  Promise&lt;sequence&lt;VRDisplay&gt;&gt; getVRDisplays();
-  readonly attribute FrozenArray&lt;VRDisplay&gt; activeVRDisplays;
-  readonly attribute boolean vrEnabled;
+interface VR : EventTarget{
+  Promise&lt;sequence&lt;VRDisplay&gt;&gt; getDisplays();
+  readonly attribute FrozenArray&lt;VRDisplay&gt; activeDisplays;
+  readonly attribute boolean isEnabled;
 };
 </pre>
 
-### Attributes ### {#navigator-attributes}
+### Attributes ### {#vr-attributes}
 
-<dfn method for="Navigator" id="navigator-getvrdisplays-attribute">getVRDisplays()</dfn>
+<dfn method for="VR" id="vr-getdisplays-attribute">getDisplays()</dfn>
 Return a Promise which resolves to a list of available {{VRDisplay}}s. The Promise MUST be rejected if the {{Document}} object is inside an iframe that does not have the {{allowvr}} attribute set.
 
-<dfn attribute for="Navigator" id="navigator-activevrdisplays-attribute">activeVRDisplays</dfn>
-{{activeVRDisplays}} includes every {{VRDisplay}} that is currently presenting.
+<dfn attribute for="VR" id="vr-activedisplays-attribute">activeDisplays</dfn>
+{{activeDisplays}} includes every {{VRDisplay}} that is currently presenting.
+
+<dfn attribute for="VR" id="vr-isenabled-attribute">isEnabled</dfn>
+The {{isEnabled}} attribute's getter must return true if the context object is allowed to use the feature indicated by attribute name {{allowvr}} and VR is supported, and false otherwise.
+
+### Events ### {#vr-events}
+
+User agents implementing this specification MUST provide the following new DOM events. The corresponding events must be of type {{VRDisplayEvent}} and must fire on the {{vr}} object. Registration for and firing of the events must follow the usual behavior of DOM4 Events.
 
-<dfn attribute for="Navigator" id="navigator-vrenabled-attribute">vrEnabled</dfn>
-The {{vrEnabled}} attribute's getter must return true if the context object is allowed to use the feature indicated by attribute name {{allowvr}} and VR is supported, and false otherwise.
+<dfn event for="VR" id="vrdisplayconnect-event">vrdisplayconnect</dfn>
```

Comment:
https://w3ctag.github.io/design-principles/#casing-rules by @domenic covers some but not tense. I think past tense is generally not used for events. We use past sense for promise-state attributes.